src/metadataGeneration/initializer-value.ts

```diff
@@ -43,10 +43,12 @@ export const getInitializerValue = (initializer?: ts.Expression, type?: Tsoa.Typ
       const objectLiteral = initializer as ts.ObjectLiteralExpression;
       const nestedObject: any = {};
       objectLiteral.properties.forEach((p: any) => {
-        nestedObject[p.name.text] = getInitializerValue(p.initializer);
+        nestedObject[p.name.text] = getInitializerValue(p.initializer, typeChecker);
       });
       return nestedObject;
     default:
-      return;
+      const symbol = typeChecker.getSymbolAtLocation(initializer);
+      const extractedInitializer = symbol && symbol.valueDeclaration && (symbol.valueDeclaration as any).initializer;
```

Comment:
Let's avoid breaking the type checking with any here. Something like this: 
```suggestion
      const hasInitializer = (node: ts.Node) node is ts.HasInitializer => node.hasOwnProperty('initializer');
      const extractedInitializer = symbol && symbol.valueDeclaration && hasInitializer(symbol.valueDeclaration) && symbol.valueDeclaration.initializer;
```

Reply:
Ok, I have fix it :smile: